vlingo-eventjournal/src/main/resources/db/migration/V1__EventJournal.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE vlingo_event_journal (
+  id                 UUID PRIMARY KEY,
+  event_timestamp    BIGINT       NOT NULL,
+  event_data         JSONB        NOT NULL,
+  event_metadata     JSONB        NOT NULL,
+  event_type         VARCHAR(256) NOT NULL,
+  event_type_version INTEGER      NOT NULL,
+  event_stream       VARCHAR(128) NOT NULL,
+  event_offset       INTEGER      NOT NULL
```

Comment:
I know what you mean by `event_stream` and `event_offset` and that is not logically wrong, but the typical "event sourcing" naming is `stream_name` and `stream_version` because stream normally is named and version explains that each event with that `stream_name` represents a different version of that stream. From a pure streaming perspective I think that `stream_name` and `stream_position` describe it better because it's more of a position or order(ing) than it is a version. Since multiple events can be written as a result of a single command, I find "version" to be misleading because I tend to consider all of the events cause by that command to be the "version" because they are all required to represent "what happened" then. But I sort of consider the "traditional" naming to have merit at least historically. Anyway, just informational.

Reply:
I think it makes sense to do this change, but it affects the implementation in symbio-jdbc, so maybe we can patch this in symbio-jdbc before publishing the next version and then I can change the example :D.

It will take minutes.

Reply:
Sure, no problem. I have been delayed getting release `0.7.4` out. When you have made these changes let me know and I will make the release after that.